### DISCUSSION TITLE
Show upvoted submissions (and comments) on profile page

### DISCUSSION THREAD

**Discussion by keybits** [likeandscribe/frontpage#189] [Ideas]
Similar to HN, it would be nice to see my ~~/ another user's~~ upvoted submissions and comments, something like:
<img width="410" alt="image" src="https://github.com/user-attachments/assets/67ca9712-1b92-4c4e-ab49-7100ad039ed4">

**Comment by WillCorrigan** [comment]
> Similar to HN, it would be nice to see my / another user's upvoted submissions and comments, something like: <img alt="image" width="410" src="https://private-user-images.githubusercontent.com/809823/381554067-67ca9712-1b92-4c4e-ab49-7100ad039ed4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyOTI4OTQsIm5iZiI6MTczMDI5MjU5NCwicGF0aCI6Ii84MDk4MjMvMzgxNTU0MDY3LTY3Y2E5NzEyLTFiOTItNGM0ZS1hYjQ5LTcxMDBhZDAzOWVkNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAzMFQxMjQ5NTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMzYxZTZlYzU5YzhjMzIxY2YxNTBhODg0MWQ4MDFhMGUyZWI4MjM1MzBiMzAyOThmYjYxMWUwNGViMDY3ZjIxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-L1122tCIbbLi8N-AS6Vho7mfjLUPgz83KxtxMAw7cw">

I suppose this is all readily available on things like the atproto-browser since all the pds stored things are public but does this not become a vehicle for someone to be harassed because they upvoted something?

Is there some benefit to seeing someone's upvoted posts?

**Reply by WillCorrigan** [reply]
Given that, seeing your own is probably a good addition.

**Reply by keybits** [reply]
I suppose the main use case is to see my own upvoted items as a reminder of what I liked.

So I guess the request is just to see my own (which is how HN works).

**Comment by WillCorrigan** [comment]
Coming back to this, would this be better replaced with bookmark functionality? Or would you prefer to have both?

**Reply by keybits** [reply]
hmm - for me personally, I keep bookmarkds separately.

Being able to find submissions and comments I upvoted on a site is all I really want.
